lib/order.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'menu' 
+class Order
+  attr_reader :basket
+
+  def initialize(menu)
+    @menu = menu
+    @basket = {}
```

Comment:
Storing basket items as a hash with their quantity is good, I didn't think to do this